File: Python/SpinningWords/MySolution.py
def spin_words(sentence):
    result = ""
    for word in sentence.split(" "):
        result += word[::-1] if len(word)>=5 else word
        result += " "
    return result.strip()

# Original kata: https://www.codewars.com/kata/reviews/55401901b8a14241e4000048/groups/6282751e90fd720001a46476
# My solution: 

File: CS/NameShuffler/MySolution.cs
using System;

public class Kata
{   
  public static string NameShuffler(string str)
  {
    string[] result = str.Split(' ');
    return $"{result[^1]} {result[0]}";
  }
}

// Original kata: https://www.codewars.com/kata/559ac78160f0be07c200005a
// My solution: https://www.codewars.com/kata/reviews/5991c1633c5b64d866000025/groups/67c183a1e8bcb54cdab43bf1
